src/main/java/uk/gov/hmcts/reform/sscs/healthcheck/EvidenceShareTopicConsumerHealthIndicator.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.reform.sscs.healthcheck;
+
+import java.time.Duration;
+
+import com.azure.messaging.servicebus.ServiceBusClientBuilder;
+import com.azure.messaging.servicebus.ServiceBusException;
+import com.azure.messaging.servicebus.ServiceBusReceiverClient;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+
+public class EvidenceShareTopicConsumerHealthIndicator implements HealthIndicator {
+
+    @Value("${azure.servicebus.hearings.connection-string}")
+    private String connectionString;
+
+    @Value("${amqp.topic}")
+    private String topicName;
+
+    @Value("${amqp.subscription}")
+    private String subscriptionName;
+
+    public Health health() {
+//        remove if statement once ready to release to prod
```

Comment:
I feel this is not needed/ complex. Spring JMS already provides a health indicator for JMS. You just need to add it to liveness. cc @nitinprabhuhmcts

Reply:
@adusumillipraveen  I agree it's getting complicated(I have got a branch in my local which works) but yeah not worth having this in favour of #4103 
Had a chat with @gemmatalbot and she is happy with it.
@gemmatalbot  Please close this PR and add a comment to the ticket thanks